docs/wallets/spv.md

```diff
@@ -0,0 +1,60 @@
+
+Another benefit offered by SPV is an extreme reduction in the time required for a brand new wallet to become operational, offering a huge improvement to the user experience.
+
+
+## How does SPV work?
+
+At the start of every block added to the Decred blockchain is 180 bytes of data called the [block header](../advanced/block-header-specifications.md). The block header describes key information about the block including the hash of the block, the merkle root (the sum of all the transaction hashes in the block), and the nonce calculated by the proof-of-work miners. A predetermined filter is also created for every block, based on the all of the transactions within the block. 
+
+When an SPV wallet initialises it will connect to the Decred network using peer-to-peer connections, and it will download the full set of headers and filters. It will then validate the header chain to ensure that the chain and its proof-of-work are valid. Once this is complete, the wallet will use the filters to locally identify which blocks contains owned transactions without uploading any private data to remote nodes. The wallet can then use the peer-to-peer network to download these blocks, scan them for relevant transactions and select these to update personal transaction history and balance. 
+
+
+## How is this different from a "light" wallet?
+
+"Light" wallets such as [Exodus](https://www.exodus.io/) or [Atomic](https://atomicwallet.io/) already bring some of the benefits of an SPV wallet - for example these wallets allow sending and receiving Decred transactions with a minimal start-up time and without downloading the full chain. Light wallets certainly provide a level of convenience but there are often subtle consequences because they depend upon an API provided by a centralised service:
+
+- Light wallets depend on a remote service to watch owned addresses and provide notifications when transactions are received by the addresses. Notifications could be missed or incorrect.
+
+- Users will often be required to upload an extended public key to the central server so it is aware of all of owned addresses, potentially compromising the users privacy.
+
+- Light wallets do not validate the information they receive by checking the blockchain directly - they have to blindly trust the information provided by the central server.
+
+These concerns do not apply to SPV wallets because they connect directly to the decentralised peer-to-peer Decred network, they upload no private data to remote nodes, and they discover their own transactions by inspecting the blockchain directly.
```

Comment:
Runon sentence. Suggest reword to:

> These concerns do not apply to SPV wallets because they connect directly to the decentralised, peer-to-peer Decred network. They upload no private data to remote nodes and they discover their own transactions by inspecting the blockchain directly.